lib/progressTracker.js

```diff
@@ -156,7 +161,7 @@ function asRow (name, stat) {
   ]
 }
 
-function asColor (color, row) {
+function asColor (chalk, color, row) {
```

Comment:
A nicer way to do it is to pass chalk.color directly as a function. It would probably be nicer.